docs/app/pages/components/components-sections/tree-view/tree-view/tree-view.component.html

```diff
@@ -56,6 +56,10 @@
     Full documentation on how to implement all the features listed above, and more, can be found on the <a href="https://angular2-tree.readme.io/docs/">component's website</a>.
 </p>
 
+<p>The angular- tree-component 8.5.0 introduced a dependency on mobx 5 which now requires the use of proxy objects and this is not compatiable with IE11.
+   To correct the issue use  the last IE11-compatible version of angular-tree-component 8.4.1.
```

Comment:
I don't think we need to go into this much detail, just something like "Note: The latest version of angular-tree-component that is compatible with Internet Explorer 11 is 8.4.1."

Also please move this down to the end of the section and use blockquote styling.